app/assets/stylesheets/application.css

```
@@ -0,0 +1 @@
+
```

Em xóa thông tin có sẵn trong ngày thì css có nhận trong dự án k em. Ví dụ 2 dòng này
```suggestion
 *= require_tree .
 *= require_self
```

---

dạ không nhận